sdk/keyvault/keyvault-keys/src/index.ts

```diff
@@ -148,6 +147,86 @@ export {
   logger
 };
 
+/**
+ * Formats the received options into RequestOptions
+ */
+function toRequestOptionsBase(options?: CreateKeyOptions) {
```

Comment:
missing return on this function, is lint not turned on?

Reply:
lint is turned on, I'll investigate.